### ISSUE TITLE
Implement search page title fix

### ISSUE THREAD

**Issue by nichgray** [CDRH/austen#72]
Ideally we should get the search term into the page title. Review Orchid changes and see if they can be ported over.

**Comment by nichgray** [comment]
This should be resolved with https://github.com/CDRH/austen/pull/71/changes/753f2737df92acde7c3c9e2cd078fa151b379030 (and subsequent tinkering).
